benchmarks/Cargo.toml

```diff
@@ -38,9 +38,6 @@ starknet-crypto = "0.6"
 tokio = { version = "1", features = ["full"] }
 tracing = { version = "0.1", default-features = false }
 tracing-subscriber = { version = "0.3", features = ["env-filter"] }
-zeth-guests = { git = "https://github.com/risc0/zeth", branch = "capossele/bump-risc0-f265059f", package = "zeth-guests" }
-zeth-lib = { git = "https://github.com/risc0/zeth", branch = "capossele/bump-risc0-f265059f", package = "zeth-lib" }
-zeth-primitives = { git = "https://github.com/risc0/zeth", branch = "capossele/bump-risc0-f265059f", package = "zeth-primitives" }
```

Comment:
What's the reason for removing zeth?

Reply:
I've disabled the benchmark apps workflow in CI for now since it takes a very long time to run and currently fails because of other issues in CI. I think we can add zeth back at some point, perhaps when zeth has released crates. I'd like to be able to make breaking changes to our code without having to also keep zeth up to date.